### PR TITLE
Portfolio Website: Setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by Shisui6**
## Create Navbar and Intro Section
- Set up linters for HTML and CSS
- Add HTML and CSS files
- Add a descriptive README.md
- Add images 
- Create and style navbar and intro section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <link rel="stylesheet" href="./resources/CSS/reset.css">
+  <link rel="stylesheet" href="./resources/CSS/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+  <header>
+    <p>My Logo</p>
```
Comment (Ugboaja-Uchechi) [inline index.html:16]: - [ ] Great job 👍 . Kindly wrap your logo in an `a` tag 🙏 . These are for three reasons: convention, semantics, and user experience(UX).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <button type="button"><img src="./resources/images/Union.svg" alt="menu"></button>
```
Comment (Ugboaja-Uchechi) [inline index.html:17]: - [ ] Kindly wrap your `hamburger menu` in a `nav` tag 🙏 . The `nav` tag is for semantic reasons and the right convention and user experience for mobile devices.

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Shisui6,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sajid-munawar — APPROVED** [review]
### Hi @Shisui6 , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
> ## [Optional] suggestions:
> * [ ]  Please check my inline comments.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @sajid-muwanar in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**